src/libraries/ProtocolFeeLibrary.sol

```diff
@@ -32,9 +41,9 @@ library ProtocolFeeLibrary {
         }
     }
 
-    // The protocol fee is taken from the input amount first and then the LP fee is taken from the remaining
-    // The swap fee is capped at 1_000_000 (100%) for cl pool and 100_000 (10%) for bin pool
-    // Equivalent to protocolFee + lpFee(1_000_000 - protocolFee) / 1_000_000
+    /// @notice The protocol fee is taken from the input amount first and then the LP fee is taken from the remaining
+    /// Equivalent to `protocolFee + lpFee(1_000_000 - protocolFee) / 1_000_000`
+    /// Hence the swap fee is capped at 1_000_000 (100%) for cl pool and 100_000 (10%) for bin pool
```

Comment:
could we remove the world `hence` (might be misleading)

As this implies that the swap fee is capped due to 
`Equivalent to `protocolFee + lpFee(1_000_000 - protocolFee) / 1_000_000`

however bin pool isn't capped to that due to this formula